utils/hct/hcttest.cmd

```diff
@@ -93,6 +93,8 @@ if "%1"=="-clean" (
 ) else if "%1"=="clang-filter" (
   set TEST_ALL=0
   set TEST_CLANG=1
+  set TEST_USE_LIT=0
+  echo Fallback to taef when use taef only options.
```

Comment:
`echo` could happen in one place when you detect that TEST_USE_LIT==0 after option parsing.